packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1944,6 +1944,8 @@ class CustomMultiChildLayout extends MultiChildRenderObjectWidget {
 /// specified height and width as possible given the parent's constraints. If
 /// [height] or [width] is null or unspecified, it will be treated as zero.
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=EHPu_DzRfqA}
```

Comment:
`SizedBox` already has an embedded widget of the week video.

Reply:
oh sorry about that! That was a merge fail on my part; I had an old flutter checkout on my computer. fixed!